bsip-0084.md

```diff
@@ -0,0 +1,195 @@
+    BSIP: 
+    Title: Elections based on non-core asset
+    Authors: Peter Conrad
+    Status: Draft
+    Type: Protocol
+    Created: 2019-10-12
+    Discussion: https://github.com/bitshares/bsips/issues/81
+
+# Abstract
+
+From the beginning, the BitShares blockchain has offered the ability to vote with the core token, and to have elected accounts govern the blockchain.
+
+For specific use-cases it can be desirable to have a similar mechanism for voting and elections where the votes are counted in relation not to BTS but to some other asset. An example might be a community of people who are interested in a specific topic.
+
+This BSIP proposes changes that will enable elections based on dedicated assets.
+
+
+# Motivation
+
+The feature has been requested from independent businesses as well as from within the community. In addition, it paves the way for a to-be-proposed change to BitAsset governance, i. e. the decoupling of BitAsset management from blockchain governance.
+
+
+# Rationale
+
+There are fundamental differences between the mechanics proposed here and the mechanics already in place for BTS-based voting.
+
+BTS balances are affected by almost every operation, due to transaction fees. The voting tokens will only be affected when they are being used.
+
+BTS is used in a multitude of ways, e. g. as collateral, as the counterpart in most active markets, as payment for workers and witnesses, as cashback for fees and so on. Contrarily, it is assumed that the primary purpose of the voting tokens will be voting. They are unlikely to be used as collateral.
```

Comment:
> Contrarily, it is assumed that the primary purpose of the voting tokens will be voting.  They are unlikely to be used as collateral.

Personally I won't assume this, but the opposite.

Reply:
Hm, interesting. Anything specific you have in mind?

IMO in order to be used as collateral, a token would need to have some intrinsic value, which I don't see in a voting token.

Reply:
A voting token would not have value if it tracks the voting weight of another token well (which is hard), in that case it is a simple utility token carries one role for another token. When we have this BSIP (token-based voting), I'd expect that people would directly vote with the tokens they already have but not create new tokens only for voting, because it would be economically much cheaper.

This is also the reason that I don't agree with the MANAGER token idea proposed in BSIP83, because the new token can't track the distribution of BTS well in the long run. That idea means to privatize committee-owned smartcoins, which is something similar to STEALTH, and would divide the community.

On the other hand, there would also be use cases that a voting token is needed, e.g. if the voting weights are time-based or derived from another token.